rust/arrow/src/compute/kernels/cast.rs

```diff
@@ -1719,20 +1791,27 @@ mod tests {
 
     #[test]
     fn test_cast_string_to_timestamp() {
-        let a = StringArray::from(vec![
```

Comment:
I recommend adding coverage here to convert from `LargeUtf8` string arrays as well - I don't think they are covered by these tests

Reply:
@nevi-me I just fixed this test. I already modified the test with that intention but I apparently tested `stringarray` twice instead of `stringarray` and `largestringarray`

Reply:
Thanks @ritchie46 !